p4proto/bfIntf/bf_chassis_manager.cc

```diff
@@ -158,17 +176,68 @@ ::util::Status BfChassisManager::ValidateAndAdd(uint64 node_id, uint32 port_id,
       LOG(INFO) << "ValidateAndAdd::kHostConfig = " << config_params.host_name();
       break;
 
+    case SetRequest::Request::Port::ValueCase::kPipelineName:
+      validate |= GNMI_CONFIG_PIPELINE_NAME;
+      config.pipeline_name = config_params.pipeline();
+      LOG(INFO) << "ValidateAndAdd::kPipelineName= " << config_params.pipeline();
+      break;
+
+    case SetRequest::Request::Port::ValueCase::kMempoolName:
+      validate |= GNMI_CONFIG_MEMPOOL_NAME;
+      config.mempool_name = config_params.mempool();
+      LOG(INFO) << "ValidateAndAdd::kMempoolName= " << config_params.mempool();
+      break;
+
+    case SetRequest::Request::Port::ValueCase::kControlPort:
+      config.control_port = config_params.control();
+      LOG(INFO) << "ValidateAndAdd::kControlPort= " << config_params.control();
+      break;
+
+    case SetRequest::Request::Port::ValueCase::kMtuValue:
+      validate |= GNMI_CONFIG_MTU_VALUE;
+      config.mtu = config_params.mtu();
+      LOG(INFO) << "ValidateAndAdd::kMtuValue= " << config_params.mtu();
+      break;
+
     default:
       break;
   }
 
   node_id_port_id_to_backend_[node_id][port_id] = validate;
-  if (validate == GNMI_CONFIG_TDI) {
-      // Required parameters are configured
-      LOG(INFO) << "Required parameters are configured, configure port via TDI";
-      LOG(INFO) << "SDK_PORT ID while validating = " << sdk_port_id;
-      RETURN_IF_ERROR(AddPortHelper(node_id, unit, sdk_port_id, singleton_port, &config));
-  }
+    if ((validate & GNMI_CONFIG_PORT_TYPE) == GNMI_CONFIG_PORT_TYPE) {
+      if (((config.port_type == PORT_TYPE_VHOST) &&
```

Comment:
Also LINK to VHOST question related to operational use...

Reply:
Sorry, I didn't understand the question here?